src/PrestaShopBundle/Controller/Admin/Sell/Order/OrderController.php

```diff
@@ -131,12 +135,73 @@ private function getOrderToolbarButtons(): array
         return $toolbarButtons;
     }
 
-    //@todo: wip
+    /**
+     * Places an order from BO
+     *
+     * @AdminSecurity("is_granted('create', request.get('_legacy_controller'))")
+     *
+     * @param Request $request
+     *
+     * @return RedirectResponse
+     */
+    public function placeAction(Request $request)
+    {
+        $summaryForm = $this->createForm(CartSummaryType::class);
+        $summaryForm->handleRequest($request);
+
+        if ($summaryForm->isSubmitted() && $summaryForm->isValid()) {
+            $formData = $summaryForm->getData();
+            try {
+                $orderId = $this->getCommandBus()->handle(new AddOrderFromBackOfficeCommand(
+                    (int) $formData['cart_id'],
+                    $this->getContext()->employee->id,
+                    $formData['order_message'],
+                    $formData['payment_module'],
+                    (int) $formData['order_state']
+                ));
+
+                return $this->redirectToRoute('admin_orders_view', [
+                    'orderId' => $orderId->getValue(),
+                ]);
+            } catch (Exception $e) {
+                $this->addFlash('error', $this->getErrorMessageForException($e, $this->getErrorMessages($e)));
+            }
+        }
+
+        return $this->redirectToRoute('admin_orders_create');
+    }
+
+    /**
+     * Renders create order page.
+     * Whole page dynamics are on javascript side.
+     * To load specific cart pass cartId to url query params (handled by javascript)
+     *
+     * @AdminSecurity("is_granted('create', request.get('_legacy_controller'))")
+     *
+     * @return Response
+     */
```

Comment:
I think it would be clearer if this action was renamed `orderFormAction` or `formPageAction`
So that the `placeAction` can be renamed `createAction` which would seem more logical considering what it does